src/engine/GroupBy.cpp

```diff
@@ -720,6 +722,11 @@ std::optional<GroupBy::ExplicitlySortedData> GroupBy::checkIfExplicitlySorted(
     return std::nullopt;
   }
 
+  // Only allow up to 5 aggregates for now (because of CallFixedSize)
```

Comment:
Add a `TODO<yourUsername>` remove this as soon as we have a better implementation for multiple aggregates.